### ISSUE TITLE
Tests for WCAG Failure F2: using changes in text presentation to convey information without using the appropriate markup or text

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#156]
http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/F2.html

A failure that occurs when a change in the appearance of text conveys meaning without using appropriate semantic markup. This failure also applies to images of text that are not enclosed in the appropriate semantic markup.

Failure Example 1: Using CSS to style the p element to look like a heading
Failure Example 2: Images of text used as headings where the images are not marked up with heading tags
Failure Example 3: Using CSS to visually emphasize a phrase or word without conveying that emphasis semantically
## Tests Procedure

For images of text:
1. Check if any images of text are used to convey structural information of the document.
2. Check that the proper semantic structure (e.g., HTML headings) is used with the text to convey the information.

For styled text that conveys information:
1. Check if there is any styled text that conveys structural information.
2.  Check that in addition to styling, the proper semantic structure is used with the text to convey the information.

Related
#75
